app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split(" ")
```

Comment:
`" "` is default value for split
```suggestion
    split_command = command.split()
```